.gitignore

```diff
@@ -20,5 +20,5 @@ node_modules
 /yarn-error.log
 _actual*.*
 _output
-/types
+types
```

Comment:
i'm a little nervous this might cause unexpected issues in the future if we have another subproject with a `types` directory. can we put the full path?

Reply:
I think /types was also targeting sub directories, that's why i experimented with it, and gave up when it didn't work. I gave up at types only 😅

Will changr

Reply:
Does anyone 😅

Reply:
Done!